src/App.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './App.css';
+import SmartQuestionsList from './view/SmartQuestionList';
+import SmartLogin from './view/SmartLogin';
+import {HashRouter, Switch, Route} from 'react-router-dom';
+import SmartCreateQuestion from './view/SmartCreateQuestion';
+
+/*const questions =  [{
```

Comment:
You should strive to remove commented out code; you anyway moved this inside the model + committed it to git so you won't lose it.